### ISSUE TITLE
not possible to use more than 4 accounts

### ISSUE THREAD

**Issue by geojak** [Haaxor1689/turtle-autologin#1]
when you have 4 accounts saved and log first time into a 5 account, on login it will flip the page and add the new account on a new page. the data is saved in the config file too.

next time you restart the client it is NOT loaded and you cant to use the arrow keys to acces the new created page with the 5 account

but it is weridly bugged because if i save an account called dummy then it gets loaded prpoerly

**Comment by Haaxor1689** [comment]
try deleting the `SET accountName` line from config.wtf and filling it out again

**Comment by Haaxor1689** [comment]
hmm it's possible that the detection of too much data isn't working correctly, can you check the length of the text in `SET accountName "..."` between the `""`? for the dummy and for what you are trying to enter?

**Comment by Haaxor1689** [comment]
ah I see so the limit is actually shorter seems like

**Comment by zmarotrix** [comment]
I'm also having this issue. If I log into a 5th account and log back out it seems to be stored just fine, but if I close the game and relaunch it, the 5th account is gone. It's also not stored in my config.wtf.

**Comment by Haaxor1689** [comment]
yes the client sadly has a hard limit on the field where I'm saving the data and while the game is running it's not checking properly if it actually has enough space to save
